helm-charts/stable/amazon-eks-pod-identity-webhook/templates/Service.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "amazon-eks-pod-identity-webhook.fullname" . }}
+  labels:
+    {{- include "amazon-eks-pod-identity-webhook.labels" . | nindent 4 }}
+  annotations:
+    prometheus.io/port: {{ .Values.service.port | quote }}
```

Comment:
This is actually a [separate port](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/main.go#L44), defaulting to `9999`

Reply:
I just kept it as the same `443` port due to the yaml file here: https://github.com/vivek-koppuru/amazon-eks-pod-identity-webhook/blob/master/deploy/service.yaml#L7

Which one should I follow?

Reply:
That file should be updated to use 9999. You could add the `--metrics-port` flag to the webhook deployment, and plumb that value down to this line too

Reply:
Sounds good